lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -203,6 +203,7 @@ <h5 class="hd hd-5">${_("Task Status")}</h5>
     <hr>
 </div>
 %endif
+%endif
 
 %if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
 <div class="running-tasks-container action-type-container">
```

Comment:
This block should only be under `if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):` condition. It was mistakenly put under another irrelevant condition due to wrong placement of `%endif`.